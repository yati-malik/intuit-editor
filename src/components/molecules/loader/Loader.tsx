import React from 'react';
import styles from './loader.module.scss';

export const Loader = () => {
    return (
        <div className={styles['loader-container']}>
            <div className={styles['spinner']}></div>
        </div>
    )
}