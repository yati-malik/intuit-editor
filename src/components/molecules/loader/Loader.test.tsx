import { render } from '@testing-library/react'
import { Loader } from './Loader'

test('renders text inside button', async () => {
    const { asFragment } = render(<Loader></Loader>)
    expect(asFragment()).toMatchSnapshot();
})