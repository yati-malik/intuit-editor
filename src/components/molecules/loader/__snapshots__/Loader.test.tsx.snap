// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders text inside button 1`] = `
<DocumentFragment>
  <div
    class="loader-container"
  >
    <div
      class="spinner"
    />
  </div>
</DocumentFragment>
`;
